ui/src/utils/prop-types.js

```diff
@@ -180,6 +180,31 @@ export const workers = shape({
   edges: arrayOf(worker),
 });
 
+export const WMWorkers = arrayOf(
+  shape({
```

Comment:
In this file, we could define what a `WMWorker` looks like and then in the component file we could say that it is an array of `WMWorker`. In other words, instead of specifying `arrayOf` in this file, we would do it in `WMWorkersTable/index.jsx `.

Reply:
what are the advantages to this method?

Reply:
Advantage: If a future view will be responsible to render a single worker manager worker, then it would be able to grab the `WMWorker` structure from here.